.travis.yml

```diff
@@ -9,18 +9,25 @@ rvm:
 script: "bundle exec rake clean spec cucumber"
 
 gemfile:
-  - gemfiles/3.2.gemfile
-  - gemfiles/4.1.gemfile
-  - gemfiles/4.2.gemfile
   - gemfiles/3.2.awsv1.gemfile
+  - gemfiles/3.2.awsv2.gemfile
   - gemfiles/4.1.awsv1.gemfile
+  - gemfiles/4.1.awsv2.gemfile
   - gemfiles/4.2.awsv1.gemfile
+  - gemfiles/4.2.awsv2.gemfile
+  - gemfiles/master.awsv1.gemfile
+  - gemfiles/master.awsv2.gemfile
 
 matrix:
   fast_finish: true
   allow_failures:
     - rvm: jruby-19mode
     - rvm: rbx-2
+  exclude:
+    - gemfile: master.awsv1.gemfile
```

Comment:
> yep, i think that's what the problem is. will squash down all of my commits before merging in :P

Excellent, and rebase on top of latest master too. Thank you, @maclover7! :clap: